aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1238,12 +1318,14 @@ static inline Tensor& bmm_out_or_baddbmm_(Tensor& self_or_result, const Tensor&
 
```

Comment:
Aren't the lines above (until the beginning of the function) redundant?

Reply:
My bad, fixed now.